### PR TITLE
[HELP NEEDED] [DO NOT MERGE] The Mapper Port Hell!

### PR DESCRIPTION
**Pull request by HushTD**
As allowed by Lion, I started porting more mapper stuff from bay for us to use, or make our maps better

Ports:

https://github.com/Baystation12/Baystation12/pull/21819
Which fixes floating lights on APCs, unbreakable small walls and some other door and smoothwalling issues! - WORKING FROM TESTING

https://github.com/Baystation12/Baystation12/pull/21741
Which fixes catwalk smoothwalling and allows seeing under the catwalks like you should be able to! - WORKING FROM TESTING, MORE TESTING MIGHT BE NEEDED FROM ME

https://github.com/Baystation12/Baystation12/pull/21252
Which makes many many sprites look nicer, or fit 3/4 PErspective, Consoles, Laptops and Tablets were omitted, since Lion does not like the futuristic sprites - No real place for issues to arise

https://github.com/Baystation12/Baystation12/pull/21155 - It gives a better look to floors, adds meny new floors, and beautiful new and old improved decals for mappers 

The Last PR has an issue which I am too retarded code wise to fix. The floor tiles as an item exist, the floors as turfs exist, but due to reasons unknown, one cannot place the new floortiles items to create the new turfs, old ones work perfectly and appear to be using the same code, so here I need help!

There might be some compile issues at first, since a few floors might have changed paths, but these are hyper easy fixes!

~Resh

### CODE REVIEW

File: code/game/machinery/camera/camera.dm [hunk 1]
```diff
@@ -83,8 +90,6 @@
 			error("[src.name] in [get_area(src)]has errored. [src.network?"Empty network list":"Null network list"]")
 		ASSERT(src.network)
 		ASSERT(src.network.len > 0)
-		
-	global.camera_list += src
```
Comment (kachnov) [inline code/game/machinery/camera/camera.dm]: these are here for a reason

File: code/game/machinery/camera/camera.dm [hunk 2]
```diff
@@ -102,7 +107,6 @@
 
 
 /obj/machinery/camera/Destroy()
-	global.camera_list -= src
```
Comment (kachnov) [inline code/game/machinery/camera/camera.dm]: and this

File: code/game/objects/structures/catwalk.dm [hunk 3]
```diff
@@ -67,4 +66,35 @@
...
+		return
+	if(istype(C, /obj/item/stack/tile/mono) && !plated_tile)
+		var/obj/item/stack/tile/floor/ST = C
+		if(!ST.in_use)
+			to_chat(user, "<span class='notice'>Placing tile...</span>")
+			ST.in_use = 1
+			if (!do_after(user, 10))
+				ST.in_use = 0
+				return
+			to_chat(user, "<span class='notice'>You plate the catwalk</span>")
```
Comment (kachnov) [inline code/game/objects/structures/catwalk.dm]: grammar

File: code/modules/power/cable.dm [hunk 4]
```diff
@@ -53,25 +53,25 @@ var/list/possible_cable_coil_colours
 	return PN.draw_power(amount)
 
 /obj/structure/cable/yellow
-	color = COLOR_YELLOW
+	color = COLOR_AMBER
```
Comment (kachnov) [inline code/modules/power/cable.dm]: why `COLOR_AMBER` and not `COLOR_YELLOW`?

**Review by kachnov — CHANGES_REQUESTED** [review]

File: code/modules/power/cable.dm (hunk 4 above)
Comment (LYNXEMS) [reply code/modules/power/cable.dm]: Oh, I believe I ported the more dark colored cables by accident, but I think they look neater, and not as bright, have you ever seen neon cables?

Comment (JakeDaBoss) [reply code/modules/power/cable.dm]: Totally unnecessary.

Comment (JakeDaBoss) [reply code/modules/power/cable.dm]: It is. That's usual. Cables being bright and visible is always good.

File: code/game/objects/structures/catwalk.dm (hunk 3 above)
Comment (LYNXEMS) [reply code/game/objects/structures/catwalk.dm]: This is the correct past tense, I will add the dot

Comment (kachnov) [reply code/game/objects/structures/catwalk.dm]: its present tense and the tense is fine, I just wanted the period. Bay has a tendency to miss things like that.

File: code/modules/power/cable.dm (hunk 4 above)
Comment (LYNXEMS) [reply code/modules/power/cable.dm]: Will undo

**Comment by LYNXEMS** [comment]
Cable colours undone, but still the main issue with new floor tiles unplacable is my main concern

**Comment by JakeDaBoss** [comment]
This is why you leave port jobs to coders. You don't end up with these problems.

And if you do ask coders to do this, I'm pushing it to the back burner. Literally only an aesthetic overhaul that can be done later.

**Comment by LYNXEMS** [comment]
I mean I did 99% and only one issue is left that probably is a super quick fix, and the visual overhaul is really good, and well, it is a learning experience

**Comment by JakeDaBoss** [comment]
We've got three SCP's in development still, even with 513 released. We're not going to be spending time on this right now, no matter how trivial it might be.

**Comment by LYNXEMS** [comment]
Also coders have worse issues to focus on, so possibly helping fix one small issue whenever they have time is much better than requesting them to port everything

**Comment by LYNXEMS** [comment]
I am not saying we must do this immediately, just this PR can be kept open until someone finds time

**Review by JakeDaBoss — CHANGES_REQUESTED** [review]
Do not merge, and will not be fixed this time. Once we have some more time, we'll revisit this.

**Pull request closed without merging**